internal/backends/sqlite/metadata/registry.go

```diff
@@ -169,6 +198,33 @@ func (r *Registry) CollectionCreate(ctx context.Context, dbName string, collecti
 	return true, nil
 }
 
+// CollectionGet returns table name associated with provided collection.
+func (r *Registry) CollectionGet(ctx context.Context, dbName string, collectionName string) (string, error) {
+	db := r.p.GetExisting(ctx, dbName)
+	if db == nil {
+		return "", nil
+	}
+
+	query := fmt.Sprintf("SELECT table_name FROM %q WHERE name = ?", metadataTableName)
+
+	rows, err := db.QueryContext(ctx, query, collectionName)
+	if err != nil {
+		return "", lazyerrors.Error(err)
+	}
+	defer rows.Close()
+
+	if !rows.Next() {
+		return "", lazyerrors.New("no collection found")
```

Comment:
I guess it's not a lazy error but some other error? Maybe the easiest way is to run some test that already queries/modifies a non-existent collection to see how this should reflect it.